packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.props.ts

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import {
+    ManagedClasses,
+    ViewportPositionerClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface ViewportPositionerManagedClasses
+    extends ManagedClasses<ViewportPositionerClassNameContract> {}
+export interface ViewportPositionerUnhandledProps
+    extends React.HTMLAttributes<HTMLDivElement> {}
+
+export enum ViewportPositionerHorizontalPosition {
+    left = "left",
+    right = "right",
+    uncontrolled = "uncontrolled",
+}
+
+export enum ViewportPositionerVerticalPosition {
+    top = "top",
+    bottom = "bottom",
+    uncontrolled = "uncontrolled",
+}
+
+export enum AxisPositioningMode {
+    uncontrolled = "uncontrolled",
+    flipOutward = "flipOutward",
```

Comment:
do you think flipOutward should now be "auto" since you are using "inset" below?

Reply:
Nah, I think "flipOutward" is more descriptive.